src/components/ChatEntry.js

```diff
@@ -1,22 +1,42 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+import { useState } from 'react';
 
 const ChatEntry = (props) => {
+  const { id, sender, body, timeStamp, onLikeChange} = props;
+  const [liked, setLiked] = useState(false);
+
+  const handleLikeClick = () => {
+    setLiked((prevLiked) => !prevLiked);
+    onLikeChange(id, !liked);
+  }
+
+  const isLocal = sender === 'Vladimir';
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={`chat-entry ${isLocal ? 'local' : 'remote'}`} key={id}>
+    {/* // <div className="chat-entry local" key={id}> */}
+      <h2 className="sender">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{body}</p>
+        <p className="entry-time"><TimeStamp time={timeStamp}></TimeStamp></p>
+        {/* <span>{props.sender}</span>
+        <span>{props.body}</span> */}
+
+        {/* <span>{props.chatTime}</span> */}
+        
       </section>
+      <button className="like" onClick={handleLikeClick}>
+          {liked ? '❤️' : '🤍'}
+      </button>
     </div>
   );
 };
 
 ChatEntry.propTypes = {
```

Comment:
Yay, prop types! It would be good to add in all of the things this component needs (like id, timeStamp, and onLikeChange).